types/array.from/.editorconfig

```diff
@@ -0,0 +1,3 @@
+# This package uses tabs
+[*]
+indent_style = tab
```

Comment:
Is this necessary?

Reply:
Yes.

Otherwise Prettier will convert tabs to spaces, which I don’t want.